forecasting/setup.py

```diff
@@ -26,10 +28,17 @@
     "pandas",
     "psutil>=5.7.3,<5.9",
     "gluonts>=0.8.0",
-    "mxnet<2.0",
     f"autogluon.core=={version}",
+    f'autogluon.common=={version}',
 ]
 
+try:
+    from mxnet import __version__ as mxnet_version
+    assert vparse(mxnet_version) < vparse("2.0")
+except (ImportError, AssertionError):
+    raise ImportError("autogluon.forecasting depends on Apache MxNet v1.x. Please install a suitable version of "
+                      "MxNet before continuing with the installation.")
+
```

Comment:
Does it really support any version `<2.0`? Should we provide a lower bound as well?